### PR TITLE
Upgrade tooling, use GitHub Actions

### PR DESCRIPTION
**Pull request by kerkhofsd**

### CODE REVIEW

**Comment by kerkhofsd** [comment]
Build says no... 

@anghelutar what the reason for the `alfresco-solr6:2.0.0` image being under the `alfresco-enterprise` namespace? 
I don't think there's a distinction between communicty <-> enterprise Solr?

File: Jenkinsfile
Comment (vierbergenlars) [inline Jenkinsfile:24]: Will this only run on PRs opened by us?

Or will it also trigger on PRs created by external people? That would be dangeous.

**Review by vierbergenlars — CHANGES_REQUESTED** [review]

File: Jenkinsfile
Comment (kerkhofsd) [reply Jenkinsfile:24]: In practice we don't trigger separate builds for PR's on Jenkins anyway, hence I'll remove this. 

A bit unfortunate though that we'll run these tests only after it will be merged to master?

**Comment by anghelutar** [comment]
> Build says no...
> 
> @anghelutar what's the reason for the `alfresco-solr6:2.0.0` image being under the `alfresco-enterprise` namespace?
> I don't think there's a distinction between community <-> enterprise SearchServices?

search-services:2.0.0 is not public on alfresco's nexus, only SNAPSHOT's are public there

File: Jenkinsfile
Comment (kerkhofsd) [reply Jenkinsfile]: Actually I'm thinking about adding this again. 
PR's don't run automatically on Jenkins, but it gives to option to trigger it manually and run these tests for a PR. 

WDYT?

**Comment by kerkhofsd** [comment]
Good to go.

File: Jenkinsfile
Comment (vierbergenlars) [reply Jenkinsfile]: It is configured correctly on the Jenkins side to only trust PRs from users with write access to this repository, so you can leave it in.
I think it's better to run the integration tests on PRs than to only run them after a merge to master?

**Review by vierbergenlars — APPROVED** [review]

File: README.md
Comment (vierbergenlars) [inline README.md:148]: This is still valid, as external users will not be able to run the enterprise integration tests

Comment (kerkhofsd) [reply README.md:148]: Those are hidden behind an `-Penterprise` flag

Comment (vierbergenlars) [reply README.md:148]: Ah, okay. My approval is still valid then :smile:

**Comment by anghelutar** [comment]
Currently the root README only talks about alfresco, there is no mention of solr.
The code builds also artifacts for solr and Integration tests are also done for those. I think this should be mentioned in the root README - or there should be a link from there to alfred-telemetry-solr/README.

**Comment by kerkhofsd** [comment]
@anghelutar IMO it's no problem to have them in one project. I'd see "Alfred Telemetry" as a collection of components useful for monitoring different components in an Alfresco ecosystem. 

We indeed need to update the README according to the latest (sub)project setup.

**Pull request merged**
